app/api/helpers/calendar/ical.py

```diff
@@ -1,14 +1,12 @@
 import pytz
-from flask import jsonify
-from flask_jwt_extended import current_user
 from icalendar import Calendar, Event
 from sqlalchemy import or_
 from sqlalchemy.orm import joinedload
 
 from app.models.session import Session
 
 
-def to_ical(event, include_sessions=False, my_schedule=False):
+def to_ical(event, user_id, include_sessions=False, my_schedule=False):
```

Comment:
Should be optional, like other params. This is not a required param to export my schedule